steps/evaluation.py

```diff
@@ -68,6 +78,9 @@ def get_expectation_values_for_qubit_operator(
 
     save_expectation_values(expectation_values[0], "expectation-values.json")
 
+    if backend_specs.get("track_measurements"):
+        backend.rename_raw_measurement_data_file()
+
```

Comment:
Shouldn't this follow invocation of `backend.save_raw_measurement_data()`? If so, can you help me find it?

Note that this comment will be out of date if you decide to merge `save...` and `rename...` methods.